ga4gh/datamodel/rna_quantification.py

```diff
@@ -0,0 +1,392 @@
+        self.expressionColumnNames = [f[0] for f in _expressionColumns]
+        self.expressionColumnTypes = [f[1] for f in _expressionColumns]
+
+    def searchRnaQuantificationsInDb(
+            self, pageToken=0, pageSize=None, rnaQuantificationId=None):
+        """
+        :param pageToken: int representing first record to return
+        :param pageSize: int representing number of records to return
+        :param rnaQuantificationId: string restrict search by id
+        :return an array of dictionaries, representing the returned data.
+        """
+        sql = ("SELECT * FROM RNAQUANTIFICATION")
+        sql_args = ()
+        if rnaQuantificationId is not None:
+            sql += " WHERE id = ? "
+            sql_args += (rnaQuantificationId,)
+        sql += sqliteBackend.limitsSql(pageToken, pageSize)
+        query = self._dbconn.execute(sql, sql_args)
+        try:
+            return sqliteBackend.sqliteRows2dicts(query.fetchall())
+        except AttributeError:
+            raise exceptions.RnaQuantificationNotFoundException(
+                rnaQuantificationId)
+
+    def getRnaQuantificationById(self, rnaQuantificationId):
+        """
+        :param rnaQuantificationId: the RNA Quantification ID
+        :return: dictionary representing an RnaQuantification object,
+            or None if no match is found.
+        """
+        sql = ("SELECT * FROM RNAQUANTIFICATION WHERE id = ?")
+        query = self._dbconn.execute(sql, (rnaQuantificationId,))
+        try:
+            return sqliteBackend.sqliteRow2Dict(query.fetchone())
+        except AttributeError:
+            raise exceptions.RnaQuantificationNotFoundException(
+                rnaQuantificationId)
+
+    def searchExpressionLevelsInDb(
+            self, rnaQuantId, pageToken=0, pageSize=None, expressionId=None,
+            featureGroupId=None, threshold=0.0):
+        """
+        :param rnaQuantId: string restrict search by quantification id
+        :param pageToken: int representing first record to return
+        :param pageSize: int representing number of records to return
+        :param expressionId: string restrict search by expression id
+        :param featureGroupId: string restrict search by feature group id
+        :param threshold: float minimum expression values to return
+        :return an array of dictionaries, representing the returned data.
+        """
+        sql = ("SELECT * FROM EXPRESSION WHERE "
+               "rna_quantification_id = ? "
+               "AND expression >= ? ")
+        sql_args = (rnaQuantId, threshold)
+        if expressionId is not None:
+            sql += "AND id = ? "
+            sql_args += (expressionId,)
+        if featureGroupId is not None:
+            sql += "AND feature_group_id = ? "
+            sql_args += (featureGroupId,)
+        sql += sqliteBackend.limitsSql(pageToken, pageSize)
+        query = self._dbconn.execute(sql, sql_args)
```

Comment:
possibly, though at this point I'd like to get user feedback on the best key to order by